package.json

```diff
@@ -166,9 +166,9 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
-    "react-test-renderer": "^16.9.0",
+    "react-test-renderer": "^16.10.2",
     "start-server-webpack-plugin": "^2.2.5",
-    "storybook-chromatic": "^2.2.2",
+    "storybook-chromatic": "^3.0.0",
```

Comment:
worth checking this still works

Reply:
Nothing seems to be broken on my check

Reply:
https://www.chromaticqa.com/build?appId=5d28eb3fe163f6002046d6fa&number=4929

looks like it still works from the build here as well ^^